src/args.rs

```diff
@@ -168,6 +176,13 @@ impl Default for ColorSetting {
     }
 }
 
+/// Possible values for the `-Z` option
+#[derive(Debug, Clone, Copy, PartialEq, Eq, ValueEnum)]
+pub enum UnstableFlags {
+    /// Allow use of experimental features
```

Comment:
Similarly, I think I would just remove this line. As this flag is completely ignored, using the libtest description will likely confuse people.